examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -153,6 +153,7 @@ struct event {
   char comm[TASK_COMM_LEN];
   uint8_t error_code;
   uint8_t stack_status;
+  int32_t kernel_stack_id;
   // instead of storing symbol name here directly, we add it to another
   // hashmap with Symbols and only store the ids here
   uint64_t stack_len;
```

Comment:
I think we should keep `stack_len` and `stack_status` fields together. `stack_len` should be reduced to `uint32_t` to relax alignment requirements.

Reply:
? they are together. I can reduce to 32-bit. Don't care much about the sizes due to padding here, tho